src/mainwindow.cpp

```diff
@@ -2265,19 +2276,23 @@ void MainWindow::startCamera()
 
 void MainWindow::resetStatus(bool isConnect)
 {
+    bool imageRecordingEnabled = selectedCamera && selectedCamera->getType() != CameraImageType::SINGLE_IMAGE_FILE && selectedCamera->getType() != CameraImageType::STEREO_IMAGE_FILE;
```

Comment:
I think it would be clearer if the variable was imageRecordingDisabled and the value was negated so that it isn't constantly being used negated

Reply:
Done